### ISSUE TITLE
Footer content

### ISSUE THREAD

**Issue by geealbers** [gettypubs/beyond-the-printed-page#10]
Thinking more about the footer, I'd like to have:
- Subscribe email field and button (unless you were putting this in the header)
- Brief description with institutional links
- Social media/contact links, as icons

The text and links are below. Since they probably need to be in a different configuration that the three equal columns you have there now, I didn't want to drop them into the footer.html file.

A project by contributors from the <a href="http://www.getty.edu/publications/" title="Getty Publications">J. Paul Getty Trust</a>, the <a href="http://www.artic.edu/collections/books" title="Publications Department of the Art Institute of Chicago">Art Institute of Chicago</a>, the <a href="http://www.metmuseum.org/research/metpublications" title="MetPublications">Metropolitan Museum of Art</a> and elsewhere.

Twitter: https://twitter.com/pubsbliki
GitHub: https://github.com/gettypubs/beyond-the-printed-page
Mail: galbers@getty.edu

(we'll update the twitter handle and get a generic email later)

**Comment by egardner** [comment]
I can use icon fonts for the twitter/github/email links if you like.
Are we using mailchimp for email subscription?

**Comment by geealbers** [comment]
Great, and yes.

Also just realized that we're probably going to need a legal disclaimer. New text:

An independent project by contributors from the <a href="http://www.getty.edu/publications/" title="Getty Publications">J. Paul Getty Trust</a>, the <a href="http://www.artic.edu/collections/books" title="Publications Department of the Art Institute of Chicago">Art Institute of Chicago</a>, the <a href="http://www.metmuseum.org/research/metpublications" title="MetPublications">Metropolitan Museum of Art</a> and elsewhere. _Views expressed here are the authors' own. All content CC BY-NC-SA 3.0 unless otherwise noted._

**Comment by egardner** [comment]
This is still a work in progress – CSS tweaking is tedious...

**Comment by egardner** [comment]
Okay, we now have icon links as of d5bfb29

**Comment by geealbers** [comment]
Lovely!

**Comment by egardner** [comment]
Okay, new content is now up in the footer.

**Comment by geealbers** [comment]
Great! Could you make it so AIC's name doesn't break across the line? And I'd love to have the museum names stand out from the rest of the text on the homepage like they do on the other pages when the text is black with blue links.

Oh, and the green in the icons on the homepage feels off to me. Maybe just that it introduces a color that's nowhere else on the site. Just a comment though, not an issue.

**Comment by egardner** [comment]
In terms of color, I was trying to match the green in the background of the home page (which doesn't really appear elsewhere).

I've made the font slightly smaller to fix that line break (at the largest size, anyway: I can't control it as the layout reflows). Also, I added an underline to the footer home page links (only there, nowhere else) since using another text color here might be problematic. Let me know if you think this works better.
